modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js

```diff
@@ -90,6 +90,38 @@ function ManagementToolbar({
 
 	const searchButtonRef = useRef();
 
+	const updateFilterDropdownItems = () => {
+		if (!filterDropdownItems) {
+			return null;
+		}
+
+		filterDropdownItems.forEach((filterDropdownItem) => {
+			if (!filterDropdownItem.items) {
+				return null;
+			}
+
+			filterDropdownItem.items.forEach((item) => {
+				if (item.href) {
+					const urlParts = item.href.split('?');
+
+					const params = new URLSearchParams(urlParts[1]);
```

Comment:
Using `URLSearchParams` works fine, but you might want to consider using `URL` API. It seems to me like a much more elegant way to manage parameters. [See here an example how we are using it](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/index.js#L120-L135).